src/core/cff_parser.js

```diff
@@ -475,6 +475,9 @@ var CFFParser = (function CFFParserClosure() {
     parseCharString: function CFFParser_parseCharString(state, data,
                                                         localSubrIndex,
                                                         globalSubrIndex) {
+      if (!data) {
```

Comment:
Would it make sense to combine this with the `if` below, i.e., make the `if` below have the condition `if (!data || state.callDepth > MAX_SUBR_NESTING) {`?